File: cpp/epi/fibonacci.cc
#include <vector>

#include "generic_test.h"
using std::vector;

int fib(int n) {
  // TODO - you fill in here.
  return 0;
}

int main(int argc, char* argv[]) {
  std::vector<std::string> args{argv + 1, argv + argc};
  std::vector<std::string> param_names{"n"};

  return GenericTestMain(args, "fibonacci.cc", "../test_data/epi/fibonacci.tsv",
                      &fib, DefaultComparator{}, param_names);
}

File: java/epi/Fibonacci.java
package epi;

import test_framework.EpiTest;
import test_framework.GenericTest;

public class Fibonacci {

  @EpiTest(testDataFile = "../test_data/epi/fibonacci.tsv")

  public static int fibonacci(int n) {
    // TODO - you fill in here.
    return -1;
  }

  public static void main(String[] args) {
    System.exit(
        GenericTest
            .runFromAnnotations(args, "Fibonacci.java",
                new Object() {
                }.getClass().getEnclosingClass())
            .ordinal());
  }
}
